node/src/test/scala/coop/rchain/node/mergeablity/ComputeMerge.scala

```diff
@@ -61,7 +62,9 @@ trait ComputeMerge {
             baseDeploysRes <- baseDeploySources.toList.traverse(
                                runtime.processDeployWithMergeableData
                              )
-            (baseDeploys, baseMergeChs) = baseDeploysRes.unzip
+            (baseDeploys, baseMergeChs, _) = baseDeploysRes
+              .map(UserTransition.unapply(_).get)
```

Comment:
Note: Not a good practice to extract value from Option with get call. In this case we know it will always succeed but it makes bad example.

Reply:
I know that but what is another way to unzip sequence of tuples wrapped in a case class?

Reply:
I don't know anything better then explicitly deconstruct.
```scala
(baseDeploys, baseMergeChs, _) = baseDeploysRes
  .map { case UserDeployRuntimeResult(deploy, mergeable, evalResult) => (deploy, mergeable, evalResult) }
  .unzip3
```